docs/book/Makefile

```diff
@@ -0,0 +1,27 @@
+MDBOOK_VERSION := v0.4.40
```

Comment:
we could have some documentation on mdbook usage under the docs/book folder, similar to: https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/docs/book